analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -510,7 +510,7 @@ def add_arguments_to_parser(parser):
                                   action='store',
                                   dest='ctu_ast_mode',
                                   choices=['load-from-pch', 'parse-on-demand'],
-                                  default='parse-on-demand',
+                                  default=argparse.SUPPRESS,
```

Comment:
With this change we will not see what is the default value for this option. So I recommend to change the help message and insert the default value at the end of the help message like this: `phase of the analysis. (default: parse-on-demand)`.

Also it would be good to update the help message with the information that this option can be used only in CTU mode.

Also you have to update this option in the `check.py` file and please update the docs:

https://github.com/Ericsson/codechecker/blob/a6b3f785adaf50b16039814ea222ae4b6b2fc0d9/analyzer/codechecker_analyzer/cmd/check.py#L511

https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#check
https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#analyze

Reply:
I always forget the check counterpart, thanks!

Reply:
This option is not updated in the check command:
https://github.com/Ericsson/codechecker/blob/eb281005a1eb1ac8be760f1777593d4d9714d2d2/analyzer/codechecker_analyzer/cmd/check.py#L518